docs/docs/mail.md

```diff
@@ -61,9 +63,9 @@ Conclave Mail provides various features that are useful when building secure app
 
 ### Encryption
 
-The encryption key used by an enclave is private to that enclave. The encryption key is stable across system restarts 
-and enclave upgrades. So, enclaves can decrypt any message encrypted and delivered by older clients. The format
-uses the respected Noise protocol framework with AES-GCM and SHA-256.
+The encryption key used by an enclave is private to that enclave. The enclave uses a random session key which 
+changes each time the enclave restarts. Enclaves can decrypt any message encrypted and delivered by older clients. 
```

Comment:
Delete the `Enclaves can decrypt any m` sentence

Reply:
Done. Also, shifted the last sentence to the top for a better flow.